### PR TITLE
Add compat data for NodeList

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/NodeList

### CODE REVIEW

File: api/NodeList.json [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "length": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NodeList/length",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: Should be `true`.

File: api/NodeList.json [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+            },
+            "chrome": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: Should be `"1"`.

File: api/NodeList.json [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: Should be `true`.

File: api/NodeList.json [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "entries": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NodeList/entries",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: This should also be `"51"`.

File: api/NodeList.json [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "forEach": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NodeList/forEach",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: 51

File: api/NodeList.json [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "item": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NodeList/item",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: true

File: api/NodeList.json [hunk 7]
```diff
@@ -0,0 +1,340 @@
...
+            },
+            "chrome": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: 1

File: api/NodeList.json [hunk 8]
```diff
@@ -0,0 +1,340 @@
...
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: true

File: api/NodeList.json [hunk 9]
```diff
@@ -0,0 +1,340 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "keys": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NodeList/keys",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: 51

File: api/NodeList.json [hunk 10]
```diff
@@ -0,0 +1,340 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "values": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/NodeList/values",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/NodeList.json]: 51

File: api/NodeList.json (hunk 1 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 2 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 3 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 4 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 6 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 7 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 8 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 9 above)
Comment (maboa) [reply api/NodeList.json]: 👍

File: api/NodeList.json (hunk 10 above)
Comment (maboa) [reply api/NodeList.json]: 👍

**Comment by maboa** [comment]
Those changes should be made now :)

File: api/NodeList.json (hunk 5 above)
Comment (jpmedley) [reply api/NodeList.json]: I think you missed one.

**Pull request merged**